test/integration/import_sb2.js

```diff
@@ -20,7 +20,7 @@ test('default', t => {
 
     // Create runtime instance & load SB2 into it
     const rt = new runtime();
-    attachTestStorage(rt);
```

Comment:
Strictly speaking, this call is incorrect: `attachTestStorage` expects a VM, not a runtime. It was working anyway because both `VirtualMachine` and `Runtime` have a method called `attachStorage` and that's all `attachTestStorage` was calling, but IMO moving the attachment out is safer and simpler than making `attachTestStorage` officially accept both types.